frame/collective/src/lib.rs

```diff
@@ -195,11 +197,12 @@ mod weight_for {
 		new_count: impl Into<Weight>,
 		proposals: impl Into<Weight> + Copy,
 	) -> Weight {
-		db.reads_writes(1, 1) // read + write members
-			.saturating_add(db.writes(2)) // set prime
-			.saturating_add(db.reads_writes(proposals.into() + 1, proposals.into())) // update votes
-			.saturating_add(old_count.into().saturating_mul(29_000_000)) // M
-			.saturating_add(new_count.into().saturating_mul(100_000)) // N
+		db.reads_writes(1, 1) // mutate `Members`
+			+ db.writes(1) // set `Prime`
+			+ db.reads(1) // read `Proposals`
+			.saturating_add(db.reads_writes(proposals.into(), proposals.into())) // update votes (`Voting`)
```

Comment:
This saturating_add only saturate db.reads(1) and this term:

```rust
db.reads_writes(1, 1) // mutate `Members`
			+ db.writes(1) // set `Prime` THIS addition is not saturated.
			+ db.reads(1) // read `Proposals` THIS also
			.saturating_add(db.reads_writes(proposals.into(), proposals.into()))
```

Reply:
My brain for some inscrutable reason thought this meant:
```rust
saturating_add(db.reads_writes(proposals.into(), proposals.into(), db.reads_writes(1, 1) + db.writes(1) + db.reads(1))
```

Reply:
yes can put parenthesis for `(db.reads_writes(1, 1) + db.writes(1) + db.reads(1))` to do so